projects/storefrontlib/src/lib/product/components/product-list/product-grid-item/product-grid-item.component.scss

```diff
@@ -1,109 +1,71 @@
 @import 'theme';
-@import 'functions';
-@import 'mixins';
 
-$cx-search-grid-image-width: 100% !default;
-$cx-search-grid-image-align: center !default;
-$cx-search-grid-image-trbl-padding: 20px 20px 20px 20px !default;
-$cx-search-grid-image-sm-padding-top: 1.25rem !default;
-
-$cx-search-grid-name-trbl-margin: 0 0 25px 0 !default;
-$cx-search-grid-name-color: 'dark' !default;
-$cx-search-grid-name-hover-color: 'primary' !default;
-$cx-search-grid-name-align: center !default;
-$cx-search-grid-name-height: 2.4em !default;
-
-$cx-search-grid-name-after-background-color: 'inverse' !default;
-$cx-search-grid-name-after-width: 100% !default;
-$cx-search-grid-name-after-height: 50px !default;
-$cx-search-grid-name-after-background: none repeat scroll 0% 0% !default;
-
-$cx-search-grid-name-before-bottom: 0 !default;
-$cx-search-grid-name-before-right: 0 !default;
-
-$cx-search-grid-price-trbl-margin: 0 0 25px 0 !default;
-$cx-search-grid-price-align: center !default;
+.cx-product-image-container {
+  display: var(--cx-display, block);
+  text-align: var(--cx-text-align, center);
+}
 
-$cx-search-grid-price-old-color: 'secondary' !default;
-$cx-search-grid-price-old-trbl-margin: 0 0 0 0 !default;
-$cx-search-grid-price-new-color: 'primary' !default;
-$cx-search-grid-price-new-trbl-margin: 0 0 25px 5px !default;
+.cx-product-image {
+  width: var(--cx-width, 100%);
+  padding: var(--cx-padding, 20px);
+  display: var(--cx-display, block);
+}
 
-.cx-product {
-  &-search-grid__image--container {
-    display: block;
-    text-align: $cx-search-grid-image-align;
+.cx-product-name {
+  @include type('5');
+  text-align: var(--cx-text-align, center);
+  display: var(--cx-display, block);
+  margin: var(--cx-margin, 0 0 25px 0);
+  color: var(--cx-g-color-dark);
+  text-decoration: var(--cx-text-decoration, none);
+  height: var(--cx-height, 2.4em);
+  overflow: var(--cx-overflow, hidden);
+  position: var(--cx-position, relative);
+
+  &:before {
+    bottom: var(--cx-bottom, 0);
+    right: var(--cx-right, 0);
+    position: var(--cx-position, absolute);
+    content: '…';
   }
 
-  &-search-grid__image {
-    width: $cx-search-grid-image-width;
-    padding: $cx-search-grid-image-trbl-padding;
-    display: block;
-
-    @include media-breakpoint-down(sm) {
-      padding-top: $cx-search-grid-image-sm-padding-top;
-    }
+  &:after {
+    content: '';
+    background: var(--cx-background, none repeat scroll 0% 0%);
+    background-color: var(--cx-g-color-inverse);
```

Comment:
This should be`background-color: var(--cx-background-color, --cx-g-color-inverse);`